client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -511,7 +524,7 @@ class EmailProvidersComparison extends Component {
 			translate,
 		} = this.props;
 
-		const formattedPrice = translate( '{{price/}} /mailbox /month billed monthly', {
+		const formattedPrice = translate( '{{price/}} /mailbox /month', {
```

Comment:
I think we should keep "billed monthly" in the text -- it's important information when comparing the two services as they do bill using different intervals.